src/fromager/sdist.py

```diff
@@ -232,6 +233,15 @@ def handle_requirement(
 
     return resolved_version
 
+# Helper method to check whether the .whl file is a zip file and has contents in it.
+# It will throw BadZipFile exception if any other file is encountered. Eg: index.html
+def download_wheel_check(wheels_prebuilt, wheel_url):
```

Comment:
Without the `ctx` prefix in the caller I don't know what `wheels_prebuilt` means. That's really the directory where the file is supposed to be downloaded, right? What is the similar argument called in `download_url()`?

Reply:
If I try to do `ctx.wheels_prebuilt`, it gives me syntax error for arguments. Should I pass `ctx` as parameter? or create a new `ctx` in helper function?

Reply:
Sorry, I wasn't clear. The variable name doesn't make sense. Can we rename it to `destination_dir` like the argument to `download_url()`?

Reply:
oh, okay! sure I will do that!